.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/LPGhatguy/lemur.git
 [submodule "modules/testez"]
 	path = modules/testez
-	url = https://github.com/Roblox/testez.git
+	url = https://github.com/Roblox/testez.git
```

Comment:
Can we revert this change in this PR?

Reply:
I...can try; this is an artifact of removing `PropTypes`. I'm not sure I'll be able to without borking the branch though. Submodules are weird >.<